packit_service/worker/whitelist.py

```diff
@@ -224,17 +230,19 @@ def check_and_report(
                 if event.trigger == TheJobTriggerType.pr_comment:
                     project.pr_comment(event.pr_id, msg)
                 else:
-                    job_helper = CoprBuildJobHelper(
-                        config=config,
-                        package_config=event.get_package_config(),
-                        project=project,
-                        metadata=EventData.from_event_dict((event.get_dict())),
-                        db_trigger=event.db_trigger,
-                    )
-                    msg = "Account is not whitelisted!"  # needs to be shorter
-                    job_helper.report_status_to_all(
-                        description=msg, state=CommitStatus.error, url=FAQ_URL
-                    )
+                    for job in job_configs:
```

Comment:
```suggestion
                    for job_config in job_configs:
```

Calling things what they are, helps understanding code :yum: